File: Instructor Java/Week03 Java/src/main/java/case01/HelloWorldRunner.java
package case01;
/**
 * C01HelloWorldRunner
 */
public class HelloWorldRunner implements Runnable {

    public void run() {
        var sample = new HelloWorld();
        sample.printHelloWorld();
        sample.printHelloWorld2();
        sample.printHelloWithInput("Waleed A.K.");
        sample.printHelloHandleField();
        sample.name("Waleed A.K.");
        sample.printHelloHandleField();
    }
}

File: Instructor Python/Week02/src/case01/HelloWorldRunner.py
from .HelloWorld import HelloWorld
class HelloWorldRunner():
    def run(self):
        sample = HelloWorld()
        sample.printHelloWorld()
        sample.printHelloWithInput('Waleed A.k')
        sample.printHelloWithInput(10)
        sample.printHelloHandleField()
        sample.name="Waleed A.K."
        sample.printHelloHandleField()